packages/react-notion-x/src/components/header.tsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+
+import { useHotkeys } from 'react-hotkeys-hook'
+import { getPageBreadcrumbs } from 'notion-utils'
+import { useNotionContext } from '../context'
+import { PageIcon } from './page-icon'
+import { SearchIcon } from '../icons/search-icon'
+import { cs } from '../utils'
+import { SearchDialog } from './search-dialog'
+
+export const Header: React.FC<{
+  header: React.ElementType
+}> = ({ header: CustomHeader }) => {
```

Comment:
Maybe I'm missing something but where is this CustomHeader type coming from? Obviously it requires some props.